### ISSUE TITLE
Upgrade ADM-Joukowski model to handle rated conditions

### ISSUE THREAD

**Issue by lawrenceccheung** [Exawind/amr-wind#784]
Putting this on the to-do list in case anybody has time.  The current ADM Joukowski model assumes that the turbine is always operating under design load conditions and the constant circulation assumption applies.  However, under off-design conditions the load distributions no longer match the analytical loads. 

Consequently, when the model is running above rated conditions, it will overpredict the power as in this example below:
![image](https://user-images.githubusercontent.com/15526007/217408610-54900f95-0d8b-43a3-a0b5-a7792c397504.png)
(see the [turbine model details](https://github.com/lawrenceccheung/AWAKEN_summit_setup/tree/main/turbines/GE1.79-100)) 

This is a known problem and Sorensen has a suggested fix in the publication here: 
https://wes.copernicus.org/preprints/wes-2022-108/
Basically there is an additional $S_0$ term that should be added to the $u_\theta$ azimuthal velocity distribution:

$$\frac{u_\theta}{U_0} = (\frac{q_0}{x}+S_0 x)g(x) F(x) $$

Hopefully it is a straightforward implementation.

Lawrence
